reco_utils/dataset/pandas_df_utils.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pandas as pd
+
+from reco_utils.common.constants import (
+    DEFAULT_USER_COL,
+    DEFAULT_ITEM_COL
+)
+
+
+def user_item_pairs(
+    user_df,
+    item_df,
+    user_col=DEFAULT_USER_COL,
+    item_col=DEFAULT_ITEM_COL,
+    user_item_filter_df=None,
+    shuffle=True
+):
+    """Get all pairs of users and items data.
+
+    Args:
+        user_df (pd.DataFrame): User data containing unique user ids and maybe their features.
+        item_df (pd.DataFrame): Item data containing unique item ids and maybe their features.
+        user_col (str): User id column name.
+        item_col (str): Item id column name.
+        user_item_filter_df (pd.DataFrame): User-item pairs to be used as a filter.
+        shuffle (bool): If True, shuffles the result.
+
+    Returns:
+        (pd.DataFrame) All pairs of user-item from user_df and item_df, excepting the pairs in user_item_filter_df
```

Comment:
the format we are using in the other files is something like:
```
pd.DataFrame: All pairs...
```

Reply:
Updating in the next commit